Formula/execline.rb

```diff
@@ -17,7 +17,6 @@ class Execline < Formula
   def install
     system "./configure",
       "--prefix=#{prefix}",
-      "--disable-static",
```

Comment:
Well, leaving the static linking disabled for `execline` is not really possible, as `s6` depends on it :(

Since `s6` depends on `skalib`+`execline` lib+`execline` binaries, and `execline` in turn depends on `skalib`, the only options I still see at the moment are:

- Leave it as it is: static `skalibs` library, static `execline` library, `s6` binary statically linked against both. You can't upgrade an upstream package without upgrading the downstream packages and reinstalling.
- Package `skalibs` with `execline`, and make `s6` depend and (statically) link against libraries from the `execline` package. Same problem as above, but one package less in the game; however, a weird split.
- Build everything in one package, and don't install any libraries. Not sure how you build different tarballs in one package in Homebrew.